pkg/cli/admin/inspect/inspect.go

```diff
@@ -135,6 +143,16 @@ func (o *InspectOptions) Complete(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if o.since != 0 {
+		o.sinceInt = (int64(o.since.Round(time.Second).Seconds()))
+	}
+	if len(o.sinceTime) > 0 {
+		o.sinceTimestamp, err = util.ParseRFC3339(o.sinceTime, metav1.Now)
+		if err != nil {
+			return err
+		}
+	}
+
```

Comment:
You can't specify both, it will fail. Logs is doing that check in `Validate` you should add such check too.